samples/azure-quantum/adapting-code/adapting-qiskit-sample.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Adapting Qiskit samples to run on Azure Quantum\r\n",
+        "\r\n",
+        "If you have some experience with quantum computing or are just starting, it is likely that you will use some samples from Qiskit.org. This Jupyter notebook walks you through 4 simple steps to adapt an existing Qiskit sample to run against any of the Azure Quantum back-ends."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 1. Download a sample from Qiskit.org\r\n",
+        "Let's say you want to try the QAOA sample: https://qiskit.org/textbook/ch-applications/qaoa.html\r\n",
+        "\r\n",
+        "1. Locate the **Download as Jupyter Notebook** hyperlink on the right side of the page\r\n",
+        "2. Download the Jupyter Notebook\r\n",
+        "3. Open your Azure Quantum workspace > **Notebooks**\r\n",
+        "4. Click **My notebooks** > ...\r\n",
+        "5. Click on the **Upload new** button and select the notebook your just downloaded\r\n",
+        "6. Open the newly imported notebook"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 2. Locate the aer_simulator\r\n",
+        "Most samples are configured to run by default against the aer_simulator, which is a great way to get started.\r\n",
+        "\r\n",
+        "1. Locate the cell that has a line like the following:\r\n",
+        "**backend = Aer.get_backend('aer_simulator')**\r\n",
+        "2. Comment out that line\r\n",
+        "\r\n"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 3. Replace the aer_simulator with an Azure Quantum backend\r\n",
```

Comment:
Should we clarify that by replacing a local simulator with a cloud simulator (like ionq or rigetti) will have differences in performance and quota? For instance, there's lots of samples where you can run many passes on AER locally within seconds, but you won't be able to just replace the backend and get similar behavior.

When working with students at last hackathon it was not clear to them that AER is a local only simulator and does not submit cloud jobs, and it was not clear to them where to find cloud job info in Job Management blade.